### DISCUSSION TITLE
No module named 'dbt'

### DISCUSSION THREAD

**Discussion by kevinle7** [AltimateAI/vscode-dbt-power-user#193] [Q&A] [answered]
Showing error "An error occured while initializing the dbt project: No module named 'dbt'.

But I have dbt installed and able to run dbt models just fine.

**Comment by mdesmet** [comment, marked as answer]
You need to select the correct Python interpreter. Check the pinned issues

**Reply by kevinle7** [reply]
Awesome thank you!
